todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksActivity.java

```diff
@@ -137,4 +139,40 @@ public boolean onNavigationItemSelected(MenuItem menuItem) {
     public IdlingResource getCountingIdlingResource() {
         return EspressoIdlingResource.getIdlingResource();
     }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        // If a task was successfully added, show mSnackBar
+        if (AddEditTaskActivity.REQUEST_CODE == requestCode) {
+
+            switch (resultCode) {
```

Comment:
Displaying a text in the snackbar is UI logic. 
We could have a method in the model `handleActivityResult(result code)` that has this entire switch. And then this can be unit tested. 
Right now the snackbar string display logic is not tested in neither UI or unit tests.